### DISCUSSION TITLE
Anyone Looking for Freelance?

### DISCUSSION THREAD

**Discussion by mwood23** [rnmapbox/maps#1528] [Q&A]
Hey folks, I'm looking for someone that can help me solve https://github.com/react-native-mapbox-gl/maps/issues/1477 and develop some more features for the app I'm building, [CampsiteTonight](https://www.campsitetonight.app/). Paid and remote of course. If there's a more appropriate place to post this let me know.

**Comment by ferdicus** [comment]
Might want to try our Gitter chat as well.
